### ISSUE TITLE
Release binary package for more python versions

### ISSUE THREAD

**Issue by zou000** [wangkuiyi/recordio#48]
Currently, we only release binary packages for Python 3.5, which is the version used by TensorFlow 1.13. TensorFlow 2.0 beta uses python 3.6. We should find a better way to release packages for multiple Python versions.

**Comment by wangkuiyi** [comment]
I think Python 3.6 + TensorFlow 2.0 beta is good enough. We are a pioneering project running in Docker containers on Kubernetes. I feel that no need to support many versions of Python and TensorFlow. However, a deep immersion in the community is a critical concern for us to win the future.

**Comment by zou000** [comment]
I agree. For ElasticDL, a python 3.6 package is sufficient. I can make an
easy change to release it. We can rethink how to release multiple version
when there is a bigger user base for RecordIO package.
-- Li
